### ISSUE TITLE
Add keybindings for showing output peeks

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#117430]
I could not figure out a keyboard way to show the inline expected vs actual zone widget.
Triggering it via keyboard should move focus to the zone widget and the proper things should be read out so this is accessible

![Screenshot 2021-02-23 at 17 21 41](https://user-images.githubusercontent.com/1926584/108873625-9ad7b100-75fb-11eb-8cca-74f40e57c7d4.png)

**Comment by connor4312** [comment]
Forgot to close, this is now `ctrl+; m`, which opens the output closest to the cursor
